post_installation_configuration/cluster-tasks.adoc

```diff
@@ -45,6 +45,12 @@ include::modules/machineset-creating.adoc[leveloffset=+3]
 Use the sample MachineSet for your cloud.
 
 include::modules/machineset-yaml-aws.adoc[leveloffset=+3]
+
+AWS MachineSets support non-guaranteed xref:../machine_management/creating_machinesets/creating-machineset-aws.html#machineset-non-guaranteed-instance_creating-machineset-aws[Spot Instances],
+which can save you costs compared to On-Demand Instance prices. You can
```

Comment:
s/save you costs/save you on costs/
^ I see that the xref topic uses this wording as well, but it sounds odd to "save costs." So maybe update in both places if you agree.